### PR TITLE
add: AllTube

### PR DESCRIPTION
**Pull request by berksmbl**

### CODE REVIEW

**Comment by ivanciucdragos1** [comment]
I think you should change version from latest to 3.2.0-alpha for example

**Comment by berksmbl** [comment]
> I think you should change version from latest to 3.2.0-alpha for example

@ivanciucdragos1 

> I think you should change version from latest to 3.2.0-alpha for example

@ivanciucdragos1 Even though I disagree, I've made the change.

**Comment by ivanciucdragos1** [comment]
Pulling image dnomd343/alltube:latest
Deployment failed
(HTTP code 500) server error - Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: Temporary failure in name resolution 
i also get this for 3.2.0-alpha, what might cause the problem?

**Comment by berksmbl** [comment]
> Pulling image dnomd343/alltube:latest Deployment failed (HTTP code 500) server error - Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: Temporary failure in name resolution i also get this for 3.2.0-alpha, what might cause the problem?

@ivanciucdragos1 
Rate Limit?
I am not experiencing any problems when I check on [link.](https://deploy-preview-409--easypanel-templates.netlify.app)

**Review by ivanciucdragos1 — APPROVED** [review]
LGTM

File: templates/alltube/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      title: Remux
+      default: true
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: alltube
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: dnomd343/alltube:3.2.0-alpha
```
Comment (deiucanta) [inline templates/alltube/meta.yaml:48]: this tag is much older then latest, please revert

Comment (berksmbl) [reply templates/alltube/meta.yaml]: I got it back. @deiucanta

**Pull request merged**